packages/terra-responsive-element/src/ResponsiveElement.jsx

```diff
@@ -4,6 +4,7 @@ import ResizeObserver from 'resize-observer-polyfill';
 import 'terra-base/lib/baseStyles';
 import breakpoints from './breakpoints.module.scss';
 
+const ResponsiveElementResponsiveTo = ['window', 'parent'];
```

Comment:
hmmmm, want to rename this? maybe `dependentViewport`

Reply:
name changed!